### DISCUSSION TITLE
how to insert text into the github copilot chat window?

### DISCUSSION THREAD

**Discussion by otac0011** [microsoft/vscode-discussions#1016] [Extension Development QnA] [answered]
I want to write an extension that programmatically puts text into the Github Copilot Chat box in the extension. is it possible to identify the chatbox in VS Code?

I tried pywinauto, but I couldn't get it to work.

**Comment by isidorn** [comment, marked as answer]
https://github.com/microsoft/vscode-extension-samples/tree/main/chat-agent-sample
https://code.visualstudio.com/blogs/2023/11/13/vscode-copilot-smarter#_extensibility
